### ISSUE TITLE
Bad formatting in agenda widget 

### ISSUE THREAD

**Issue by maikol-solis** [emacs-dashboard/emacs-dashboard#339]
Hi! 

I have a problem to visualize correctly my agenda items when there is long categories in the headings.  

For example, suppose you have those two items in a file called `actionable.org`

``` org
* TODO My task
DEADLINE: <2021-11-26 Fri>
* TODO Another task
DEADLINE: <2021-11-26 Fri>
:PROPERTIES:
:CATEGORY: LONGLONGCATEGORY
:END:
```

In my dashboard, they look like this

![image](https://user-images.githubusercontent.com/4078489/143582860-ebbe3118-d2d3-4c5c-9de6-04885bdabf1c.png)

It would be great if the category, the deadline, the TODO and the item were separated by a tab (or some flexible space), to keep each one ordered like a table. 

I picture something like this

```
actionable:       2021-11-26  TODO  My task 
LONGLONGCATEGORY: 2021-11-26  TODO  Another task 
```

Is it possible? 

Best.

**Comment by jcs090218** [comment]
This is currently not possible as I aware of, but I would like to implement this feature in the future. :)

**Comment by jcs090218** [comment]
Merged #350. You should now able to align it with the variable `dashboard-agenda-prefix-format`.
